tiledb/sm/rest/rest_client.cc

```diff
@@ -343,6 +343,67 @@ void RestClient::delete_array_from_rest(const URI& uri) {
       stats_, url, serialization_type_, &returned_data, cache_key));
 }
 
+void RestClient::delete_fragments_from_rest(
+    const URI& uri, uint64_t timestamp_start, uint64_t timestamp_end) {
+  Buffer buff;
+  serialization::fragments_timestamps_serialize(
+      timestamp_start, timestamp_end, serialization_type_, &buff);
+  // Wrap in a list
+  BufferList serialized;
+  throw_if_not_ok(serialized.add_buffer(std::move(buff)));
+
+  // Init curl and form the URL
+  Curl curlc(logger_);
+  std::string array_ns, array_uri;
+  throw_if_not_ok(uri.get_rest_components(&array_ns, &array_uri));
+  const std::string cache_key = array_ns + ":" + array_uri;
+  throw_if_not_ok(
+      curlc.init(config_, extra_headers_, &redirect_meta_, &redirect_mtx_));
+  const std::string url = redirect_uri(cache_key) + "/v1/arrays/" + array_ns +
```

Comment:
Please double check with someone from cloud if this is supposed to be `v1` or `v2` or `v3`, I never properly understood the difference.